src/components/Table/Budget/BudgetLineItemTable/BudgetLineItemTableView.tsx

```diff
@@ -13,7 +13,7 @@ function getValue(obj: any, key: string[]): any {
 	if (!obj?.hasOwnProperty(key[0])) {
 		return "";
 	}
-	if (key.length == 1) {
+	if (key.length === 1) {
 		return obj[key[0]];
 	}
 	return getValue(obj[key[0]], key.slice(1));
```

Comment:
logic can be improved

Reply:
This cover most of the test cases that we need

Reply:
It is not about the what is covered .  To understand it a developer has to give a specific time which shouldn't be the case. Extracting it to a function would increase readability